RELEASE-CYCLE.md

```diff
@@ -33,4 +33,4 @@ Sometimes there is a bug or crash that can’t wait two weeks to be fixed. This
 - Create a PR against that branch.
 - Get approvals, test very very very well, merge.
 - Submit to the store.
-- Merge back into `develop` and into `master`.
+- Merge back into `trunk`.
```

Comment:
Thank you for updating this file. I wonder, though, if we should be remove it instead 🤔 It seems redundant and incomplete.

Or, it might be interesting as an overview for contributors of what to expect. 

If we decide to remove it, it might happen in a dedicated PR, though, so 👍

Reply:
We can remove it, but as you said, it should be in a dedicated PR or be a part of a cleanup PR.

Reply:
+1 to removing this file